storage/store.go

```diff
@@ -1428,13 +1432,23 @@ func (s *Store) ProposeRaftCommand(idKey cmdIDKey, cmd proto.InternalRaftCommand
 	if value == nil {
 		panic("proposed a nil command")
 	}
-	// Lazily create group. TODO(bdarnell): make this non-lazy
-	err := s.multiraft.CreateGroup(cmd.RaftID)
-	if err != nil {
+	errChan := func(err error) chan error {
 		ch := make(chan error, 1)
 		ch <- err
 		return ch
 	}
+	// If the system is shutting down and has drained, don't accept new
+	// proposals: Raft may already have stopped.
+	select {
+	case <-s.stopper.ShouldStop():
```

Comment:
we also shouldn't be seeing this one do anything unless spurious requests exist, correct?